app/middlewares/validators/users.js

```diff
@@ -0,0 +1,28 @@
+const UserModel = require('../../models/user');
+const errors = require('../../errors');
+
+exports.checkExistingEmail = async email => {
```

Comment:
This is a middleware function, it should receive `(req, res, next)` as parameters and it should also be called from the middleware chain for this route